src/AspNet.Security.OpenId/OpenIdAuthenticationMessage.cs

```diff
@@ -132,7 +132,7 @@ public string ReturnTo
         /// <summary>
         /// Gets the parameters associated with this OpenID message.
         /// </summary>
-        protected IDictionary<string, string> Parameters { get; } =
+        public IDictionary<string, string> Parameters { get; } =
```

Comment:
I'm not sure whether or not this would count as binary breaking place, I know _more restrictive_ accessibility modifier changes is, but I'm not sure about _more permissive_.

Reply:
I suppose a more appropriate route would be modifying the message SetParameter to have the ability to not prefix a key. The issue I having is all of my custom parameters would get prefixed with openid.ax